lore/settings.py

```diff
@@ -280,3 +280,4 @@ def get_var(name, default):
         'INDEX_NAME': 'haystack'
     }
 }
+HAYSTACK_SIGNAL_PROCESSOR = 'haystack.signals.RealtimeSignalProcessor'
```

Comment:
This causes the index to be updated whenever a record is saved or deleted. This closes #142, but was unavoidable because otherwise no indexing happened during tests. Anyway, it's only one line.